src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -61,7 +61,7 @@
 
 logger = logging.get_logger(__name__)
 
-_CHECKPOINT_FOR_DOC = "gpt2"
+_CHECKPOINT_FOR_DOC = "distilgpt2"
```

Comment:
This should not be changed. "gpt2" is the official checkpoint for `GPT2` model, and it is used in the docstring example for `GPT2Model` and `GPT2LMHeadModel`.

Reply:
Okay got it will change back, I had thought they wanted us to use a lower resource version as per this instruction
```
Using a small model checkpoint instead of a large one: for example, change "facebook/bart-large" to "facebook/bart-base" (and adjust the expected outputs if any)
```